docs/guides/v8-deprecations.md

```diff
@@ -21,10 +21,12 @@ embed: true
 
 The tables below show what will be removed and what are they replaced with. We also marked if there are [codemods](#ui-codemods) available.
 
-- [Deprecated Utilities](#v8-deprecations/#deprecated-utilites)
+- [Deprecated Utilites, Properties and Components in Version 8.0](#deprecated-utilites-properties-and-components-in-version-80)
+  - [Deprecated Utilities](#deprecated-utilities)
 
 ### Deprecated Utilities
 
 | Utility                                 | Substitute                                                                                                                                          |
 | --------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------- |
 | [addResizeListener](#addResizeListener) | Use the native [ResizeObserver](https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserver) utility. Example usage: [here](#addResizeListener) |
+| [Navigation](#Navigation)               | This component has been deprecated from version 8.                                                                                                  |
```

Comment:
I'd add here a bit more explanation:
This component is deprecated in v8 and will be removed in InstUI v9 due to the lack of usage.